### PR TITLE
Add instructions to create custom DNX with CoreCLR

### PR DESCRIPTION
**Pull request by richlander**
- These instructions describe how to create a customer x64 CoreCLR DNX with the CoreCLR build.
- I couldn't get crossgen to work. That's something that still needs to be addressed.

/cc @davidfowl, @jkotas

### CODE REVIEW

File: Documentation/dnx-instructions.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+	C:\coreclr-demo>dnvm install latest -arch x64 -runtime coreclr
+
+You should now have two runtimes installed. The CoreCLR one will be the default. The install process calls `dnvm use` for you as a convenience, since that's probably what you wanted. `dnvm list` will demonstrate this change in state.
+
+	C:\Users\rlander>dnvm list
+
+	Active Version           Runtime Architecture Location                       Al
+	                                                                             ia
+	                                                                             s
```
Comment (akoeplinger) [inline Documentation/dnx-instructions.md]: "Alias" again

File: Documentation/dnx-instructions.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+	04/01/2015  04:45 PM    <DIR>          ..
+	04/01/2015  04:33 PM    <DIR>          dnx-clr-win-x86.1.0.0-beta5-11447
+	04/01/2015  04:45 PM    <DIR>          dnx-coreclr-win-x64.1.0.0-beta5-11447
+	               0 File(s)              0 bytes
+	               4 Dir(s)  94,895,509,504 bytes free
+
+
+Copy and rename the X64 CoreCLR DNX. That's the one you'll target. You'll be able to see it with `dnvm list`. This time, you will need to explictly `use` this DNX in order to use it.
+
+	C:\coreclr-demo>xcopy /E /S %USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta5-11469 %USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta5-11469c
```
Comment (akoeplinger) [inline Documentation/dnx-instructions.md]: nit: it is quite hard to visually see the `c` (which I suppose stands for `custom`?), can we just append `custom` here?

File: Documentation/dnx-instructions.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+You now need to select the custom DNX as the use to 'use'. You can see the new DNX state with `dnvm list`.
+
+	C:\coreclr-demo>dnvm use 1.0.0-beta5-11469c -r coreclr -arch x64
+	C:\coreclr-demo>dnvm list
+
+	Active Version            Runtime Architecture Location                       A
+	                                                                              l
+	                                                                              i
+	                                                                              a
+	                                                                              s
```
Comment (akoeplinger) [inline Documentation/dnx-instructions.md]: "Alias" again.

File: Documentation/dnx-instructions.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+	C:\git\coreclr>del %USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta5-1
+	1469c\bin\*.ni.dll
+
+You should now have a working custom CoreCLR x64 DNX. Give it a try.
+
+	C:\coreclr-demo>dnx . run
+
+Theoretically, you should re-generate cross-gen images for your DNX with the following commands. That doesn't seem to be working right now.
+
```
Comment (jkotas) [inline Documentation/dnx-instructions.md]: Issue #227 needs to fixed for this to work.

File: Documentation/dnx-instructions.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+	C:\coreclr-demo>dnvm install latest
+
+Both commands will install a default DNX, which on Windows, is x86 and hosted on the .NET Framework CLR. You can see that with `dnvm list`.
+
+	C:\coreclr-demo>dnvm list
+
+	Active Version           Runtime Architecture Location                       Al
+	                                                                             ia
+	                                                                             s
```
Comment (richlander) [inline Documentation/dnx-instructions.md]: This what "Alias" looks like on my screen. So, currently, the markdown is a faithful representation of what dnvm produces in a standard windows prompt. It is a bit odd, but that's what it looks like. Sound good to leave it as-is?

File: Documentation/dnx-instructions.md (hunk 4 above)
Comment (richlander) [inline Documentation/dnx-instructions.md]: Thanks @jkotas. I made reference to the issue. That explains things better.

File: Documentation/dnx-instructions.md (hunk 5 above)
Comment (akoeplinger) [inline Documentation/dnx-instructions.md]: This is just a coincidence of how PS outputs the table, on my machine (with a slightly longer `Location`) it just omits the Alias column. I'd prefer to remove it, the value isn't relevant to the instructions and it looks ugly.

Comment (richlander) [inline Documentation/dnx-instructions.md]: OK. Did that. Look good? 

@akoeplinger

Comment (akoeplinger) [inline Documentation/dnx-instructions.md]: LGTM :+1:

**Comment by davidfowl** [comment]
Looks good

**Pull request merged**